packages/cli/tests/commonlib/functionValidator.ts

```diff
@@ -172,14 +187,16 @@ export class FunctionValidator {
       });
     }
 
-    console.log("Validating app service plan.");
-    const servicePlanResponse = await this.getWebappServicePlan(
-      this.subscriptionId,
-      this.rg,
-      functionObject.appServicePlanName,
-      token as string
-    );
-    chai.assert(servicePlanResponse, functionObject.appServicePlanName);
+    if (isMultiEnvEnabled) {
```

Comment:
I didn't remember in what situation we can't get app service name from config. Do we check app service plan when multi env enabled?

Reply:
1. This condition check is wrong (it should be `if (!isMultiEnvEnabled)`, I will fix it)
2. In the multi-env + bicep case (these two flags should always be enabled together), the app service plan name is not in `profile.dev.json`, so we don't check app service plan in the multi-env case.

Example `profile.dev.json`:
![Screenshot from 2021-09-17 16-40-48](https://user-images.githubusercontent.com/9698542/133752739-9f4ae40d-cf8a-4ad5-a923-efc6f2781a7d.png)